backend/src/models/User.ts

```diff
@@ -5,43 +5,41 @@ const userSchema = new Schema(
     _id: {
       // ID generated by Google Oauth, must be called _id to be queried with .findById()
       type: String,
+      unique: true,
```

Comment:
Is there any specific reason for this? I think if a variable is called `_id` mongoDB automatically sets it as `unique` and `required`, so explicitly setting this would cause a warning (?). Feel free to correct me if I'm wrong lol